src/mailroom.py

```diff
@@ -0,0 +1,93 @@
+"""Automate thankyou notes and generates donor reports."""
+
+import io
+import sys
+from sys import version_info
+
+from builtins import input
+
+donors = {
+    'Matthew Wade': [50.00, 100.00],
+    'Betty Stewart': [100.00, 50.00, 150.00],
+    'Timothy Berg': [10.00],
+    'Paul Benson': [5]}
+
+
+def get_input():
+    """Function gathers all user input and controls program flow."""
+    # Comply inputs between Python 2 and 3. Default to Python 3:
+    user_input = input
```

Comment:
This is a nifty trick, which you could even do in the global scope near the top of the page.